src/methods/sep8Send/sendActionRequiredFields.ts

```diff
@@ -0,0 +1,68 @@
+import { log } from "helpers/log";
+import {
+  Sep8ActionRequiredSendParams,
+  Sep8ActionRequiredSentResult,
+} from "types/types.d";
+
+export const sendActionRequiredFields = async ({
+  actionFields,
+  actionMethod,
+  actionUrl,
+}: Sep8ActionRequiredSendParams): Promise<Sep8ActionRequiredSentResult> => {
+  log.request({
+    title: `Sending action required fields to SEP-8 server with \`${actionMethod} ${actionUrl}\``,
+    body: actionFields,
+  });
+  const sep8ActionRequiredResult = await fetch(actionUrl, {
+    method: actionMethod,
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(actionFields),
+  });
+  const resultJson = await sep8ActionRequiredResult.json();
+
+  let validatedResponse: Sep8ActionRequiredSentResult;
+  switch (resultJson.result) {
+    case "no_further_action_required":
+      validatedResponse = {
+        result: "no_further_action_required",
+      };
+      break;
+
+    case "follow_next_url":
+      if (!resultJson.next_url) {
+        throw new Error(`Missing "next_url" parameter`);
+      }
+      validatedResponse = {
+        result: "follow_next_url",
+        nextUrl: resultJson.next_url,
+      };
+
```

Comment:
We also could make the switch cases consts (each of them is used twice) to avoid typos.

Reply:
Good point, done in https://github.com/stellar/stellar-demo-wallet/pull/164/commits/4c2da9929b65672f6660aa7a82062221e1c56f50.